pkg/resources/certificates/certificate.go

```diff
@@ -154,10 +156,8 @@ func NewCertificateForCR(m *v1alpha1.Interconnect, profileName string, issuer st
 		},
 		Spec: cmv1alpha1.CertificateSpec{
 			SecretName: m.Name + "-" + profileName + "-tls",
-			CommonName: m.Name + "." + m.Namespace + ".svc.cluster.local",
-			DNSNames: []string{
-				m.Name + "." + m.Namespace + ".svc.cluster.local",
-			},
+			CommonName: m.Name + "." + m.Namespace,
```

Comment:
I would just use m.Name here. The service is in the same namespace so does not need qualified. The service name on its own will work within the namespace and is likely to be a common way of connecting in that scope (not sure if name.namespace will work as a lookup, suspect it needs the svc.cluster.local part?).